client/modules/IDE/components/TextOutput.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 class TextOutput extends React.Component {
   componentDidMount() {
-
+    document.getElementById('canvas-sub').focus();
```

Comment:
instead of using `document.getElementById()`, use [React refs](https://facebook.github.io/react/docs/more-about-refs.html)